src/systemtest/resources/sinceBeginningDateRange/expected/summary.json

```diff
@@ -30,7 +32,8 @@
       },
       "branch": "master",
       "displayName": "reposense/testrepo-Charlie[master]",
-      "outputFolderName": "reposense_testrepo-Charlie_master"
+      "outputFolderName": "reposense_testrepo-Charlie_master",
+      "zoneId": "Asia/Singapore"
```

Comment:
Just a question, why we need zoneId in `summary.json`? Can set the attribute not to display when generating the JSON.

Reply:
Without the zoneId, the dates used in generating the report can get ambiguous, especially since the dates will be reported differently for different time zones.

Reply:
But now, we do not support a different `zoneId` for different config, right? Even we need to pass it to frontend, there is no need to write it again and again in `summary.json`?

Reply:
@dcshzj Can you help answer this question? Why we cannot set it as a global attribute?

Reply:
My apologies, I misunderstood the original intent. I moved it out to a global attribute.